packages/app-project/src/screens/ClassifyPage/components/WorkflowAssignmentModal/index.js

```diff
@@ -1 +1 @@
-export { default } from './WorkflowAssignmentModalConnector'
\ No newline at end of file
+export { default } from './WorkflowAssignmentModalContainer'
```

Comment:
A small thing: The component naming here implies that it’s using the [container pattern](https://www.freecodecamp.org/news/react-superpowers-container-pattern-20d664bdae65/), but it isn’t any more.

Reply:
If the container pattern still makes sense for this component, then it might make sense to move the old container code into a hook.

Reply:
Good point! I'll rename